pynumdiff/utils/utility.py

```diff
@@ -1,69 +1,5 @@
-"""
-All kinds of utilities
-"""
-import os
-import sys
-import copy
+import os, sys, copy, scipy
 import numpy as np
-import scipy
-
-
-def get_filenames(path, contains, does_not_contain=('~', '.pyc')):
-    """
-    Create list of files found in given path that contain or do not contain certain strings.
```

Comment:
We have `grep -IR . -e "string"` for this, and you're not using this one anywhere, so I think it should go away.